app/src/main/java/me/capcom/smsgateway/modules/receiver/parsers/MMSRetrieveParser.kt

```
@@ -0,0 +1,383 @@
+            var contentId: String? = null
+            while (buf.position() < headersEnd) {
+                val headerCode = buf.get().toInt() and 0xFF
+                when (headerCode) {
+                    0x8E -> contentLocation = readTextString(buf) // Content-Location
+                    0xC0 -> contentId = readTextString(buf)       // Content-ID
+                    else -> {
+                        // Unknown, try skipping as text-string.
+                        buf.position(buf.position() - 1)
+                        val name = readTextString(buf)
+                        val _value = readTextString(buf)
+                        if (name.isBlank()) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyIfBlock)